src/docs/_sass/_docs-block.scss

```diff
@@ -38,6 +39,13 @@
         box-shadow: black 0 0 10px 0;
       }
     }
+
+    // This is a hotfix for animation that is triggering
+    // repaints of elements with border radius (avatars for example)
+    &--animation-container {
```

Comment:
link to http://crbug.com/157218 so that it's easy to check if this can be removed